bublik/urls.py

```diff
@@ -5,6 +5,11 @@
 from django.contrib import admin
```

Comment:
A bit different style of referencing to issue is used in the repository. Please, follow existing style.